models/labels/trader_dex_diversity/ethereum/trader_dex_diversity_ethereum.sql

```diff
@@ -0,0 +1,34 @@
+{{
+    config(
+        alias='trader_dex_diversity_ethereum',
+    )
+}}
+
+with
+ trader_dex_diversity as (
+    select
+        count(distinct project) as dex_diversity,
+        taker as address
+    from (
+        select taker, project
+        from {{ ref('dex_aggregator_trades') }}
+        where blockchain = 'ethereum' and block_time > (current_date - interval '30 days')
+        UNION ALL
+        select taker, project
+        from {{ ref('dex_trades') }}
+        where blockchain = 'ethereum' and block_time > (current_date - interval '30 days')
+    )
+    group by taker
+ )
+
+select
+  array("ethereum") as blockchain,
+  address,
+  dex_diversity as name,
```

Comment:
Maybe add: Number of DEXs traded on: `dex_diversity` ?

Reply:
Let me know if the latest update to the name is what you had in mind.

Reply:
Yes looks good! Now I'm just thinking: do you think this info will be a bit redundant with the `dex_user` model in which we print the actual names of DEXs the address has traded on? It's not a big deal we could have both, just wanted to have your opinion on this

Reply:
IMHO: Having just the number is better for visualizations as it indicates that a user uses multiple DEXs without mentioning which. Whereas, for having just the names, there are various combinations for users who use more than two DEXs and, therefore, hard to group by.